src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -345,7 +347,49 @@ bool MQ2AdvLootItemType::GetMember(MQVarPtr VarPtr, const char* Member, char* In
 		Dest.Type = pBoolType;
 		return true;
 
-	default: break;
+	case AdvLootItemMembers::Status:
+	{
+		Dest.Type = pStringType;
+
+		std::string tmp_str = "INVALID";
+
+		// Check to make sure we aren't trying to access the personal loot window and if so return INVALID.
+		if( itemList == pAdvancedLootWnd->pPLootList )
+		{
+			strcpy_s(DataTypeTemp, tmp_str.c_str());
```

Comment:
tmp_str is not necessary throughout this entire diff. There is no reason for it to exist because you can just copy the value you want directly into the destination.